webapp-django/crashstats/settings/base.py

```diff
@@ -723,7 +723,7 @@ def path(*dirs):
         },
         'postgresql': {
             'database_hostname': config(
-                'DATABASE_HOSTNAME',
+                'DATABASE_HOST',
                 'localhost'
             ),
```

Comment:
The `database_hostname` is a configman/socorro.external.postgresql.connection_context thing. That's the only place where it's called "..._hostname". This variable `socorro/webapp-django/DATABASE_HOST` (that I propose) serves two purposes:

1) Be the hostname for the django ORM
2) Be the hostname for the implementations

I think it's not worth it changing all the socorro code to call this `database_host` too. It'd affect the processors, crontabber and possibly other bits and pieces like the crashmover. 

However, I kinda like that this value is only something that matters is "django land". It needs to be "handed over" to the implementations under a different name.